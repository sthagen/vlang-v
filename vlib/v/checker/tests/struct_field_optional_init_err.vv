struct Foo {
	bar !int
	baz ?int
}

fn main() {
	_ := Foo{
		bar: 1
		baz: 1
	}
}
