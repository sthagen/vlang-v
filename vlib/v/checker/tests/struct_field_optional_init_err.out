vlib/v/checker/tests/struct_field_optional_init_err.vv:8:3: error: field `bar` is result, but initialization of result fields currently unsupported
    6 | fn main() {
    7 |     _ := Foo{
    8 |         bar: 1
      |         ~~~~~~
    9 |         baz: 1
   10 |     }
vlib/v/checker/tests/struct_field_optional_init_err.vv:9:3: error: field `baz` is optional, but initialization of optional fields currently unsupported
    7 |     _ := Foo{
    8 |         bar: 1
    9 |         baz: 1
      |         ~~~~~~
   10 |     }
   11 | }
